### ISSUE TITLE
Crash on startup 0.93 beta1 on MacOS

### ISSUE THREAD

**Issue by diogoaltoe** [pgmodeler/pgmodeler#1487]
**Bug description**
App won't start

**How to reproduce**
Click on the icon in Applications.
Allow in Security
App crashes
Try again
App crashes

**Expected behavior**
Startup

**Screenshots**
_If applicable, add screenshots to help explain your problem._

**Info about your desktop**
 - OS: macOS 
 - Version: Catalina 10.15.7
 - Window manager: PostgreSQL 12
 - pgModeler version: pgmodeler-0.9.3-beta1
 - Qt version: Qt5.13.2

**Stacktrace / Debug info**
Process: pgmodeler [52667]
Path: /Applications/pgModeler.app/Contents/MacOS/pgmodeler
Identifier: br.com.pgmodeler.pgModeler
Version: 0.9.2
Code Type: X86-64 (Native)
Parent Process: ??? [1]
Responsible: pgmodeler [52667]
User ID: 501

Date/Time: 2020-10-28 14:05:38.296 -0300
OS Version: Mac OS X 10.15.7 (19H2)
Report Version: 12
Anonymous UUID: EA594EE3-797C-6A51-AA21-B2F69BA7DE96

Time Awake Since Boot: 20000 seconds

System Integrity Protection: disabled

Crashed Thread: 0

Exception Type: EXC_CRASH (SIGABRT)
Exception Codes: 0x0000000000000000, 0x0000000000000000
Exception Note: EXC_CORPSE_NOTIFY

Termination Reason: DYLD, [0x1] Library missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
Library not loaded: @loader_path/../lib/libssl.1.1.dylib
Referenced from: /Applications/pgModeler.app/Contents/Frameworks/libpq.5.dylib
Reason: image not found

**Comment by diogoaltoe** [comment]
Any one?

**Comment by cryptedx** [comment]
No, it's working fine for me. I have the same specs and versions.

**Comment by diogoaltoe** [comment]
Today I tried the whole installation process again, according to this link: https://pgmodeler.io/support/installation

This time before running the "Resolving dependencies" step, I tried to open pgmodeler.app, and it worked!

I don't know if in the other attempts it didn't work due to something with the files being copied in this step "Resolving dependencies" or in the install_name_tool commands, but it worked without performing this step.

Thanks!
